assessments/0003-tremor.md

```diff
@@ -0,0 +1,214 @@
+
+
+## Website
+
+
+| Criteria | 1 = (Is not present or requires significant work) | 2 | 3 = (is present, but needs work) | 4 | 5 = (is executed extremely well or no improvement required) |
+| ---                       | --- | --- | --- | --- | --- |
+| Branding and design  |     |     |     |   ✅  |     |
+| Case studies/social proof |  ✅    |     |     |     |     |
+| Maintenance planning |     |   ✅  |     |     |     |
+
+**Comments**
+
+**Branding and design:**
+    - On the whole, the Tremor docs site looks professional, if a little plain. 
+    - Because of the way the site deploys (from different repositories via github pages), navigation is disjointed: if you're in the RFCs, you don't have the full site's navigation available to you and clicking the logo takes you back to https://rfcs.tremor.rs, with similar behavior occuring in the docs. This is a bad user experience.
+    - Tremor could do a bit more selling on its homepage: 
+        - What are the project's key features that make it different from other event processing systems? 
+        - Are there any known use cases you can talk about or use the logos of? 
+        - Have you given any talks on Tremor, or is there other (particularly video) content you can showcase about the project?
+
+**Case studies/social proof**: None available. 
+    - Case studies/talks/generally showing that other projects use Tremor is one of the most powerful ways to gain new users. People love real-world examples of a codebase in action.
+    - Because Tremor was developed by Wayfair, we suggest documenting Wayfair's use as a "use case" or case study, and then adding a section on the website which allows people to submit their own use cases!
+
+**Maintenance planning:** In general, we recommend that projects keep their documentation all in one repository. Tremor's documentation currently deploys from https://github.com/tremor-rs/tremor-www-main, https://github.com/tremor-rs/tremor-rfcs, and https://github.com/tremor-rs/tremor-www-docs.
+    - It's confusing for contributors who don't know where to file issues, open pull requests, or whom to tag for reviews (and where). 
+    - The easiest fix for this pain point is to deploy everything using  and Hugo. 
+    - In general, we prefer sites to deploy via Netlify. 
+
+
+**Recommendations**
+
+- Develop a case study or other content around Wayfair's use of Tremor, to give users a tangible example of what they can do with the project.
+
+- Fix navigation and improve site maintainability by putting all your documentation in one repository. 
+
+
+## Recommendations
+
+**Reorganize your documentation's information architecture:** In addition to the suggestions above, I recommend an overall reorganization of the documentation, along the lines of: 
+
+- Docs
+    - Overview (renamed from Architecture Overview)
+    - Using Tremor 
+        - .. subpages as described previously 
+    - Artefacts 
```

Comment:
```suggestion
    - Artifacts 
```

(I like the "artefact" variant too, but I don't think it's widely used any more)